guides/data/models.md

```diff
@@ -22,18 +22,18 @@ If your app can swap databases with zero refactoring, that either means:
 
 1. The underlying databases are identical in their querying and storage
 semantics (rarely the case), or
-2. You were using some lowest common denominator of querying and storage semantics for the two databases that is
+2. You were using some lowest common denominator of querying and storage semantics for the two databases that are
 equivalent across both (which means you weren't using the strengths of your original
 database)
 
 Denali is built around the assumption that different databases have different
 tradeoffs, and that you should pick the data store best suited to your use case.
 This means a good data layer should highlight the unique strengths of choice of
-data store, rather than trying to hide any differences behind a universal
+a data store, rather than trying to hide any differences behind a universal
 interface.
 
 So then why have a data layer at all for Denali? It's primarily _for addons_.
-Having some common data interface allows addons to persist and query data
+Having some common data interface allows add-ons to persist and query data
```

Comment:
"addons" is a term of art here - it's the name for a specific feature of Denali, and is not hyphenated.